examples/console_test.rs

```diff
@@ -12,24 +12,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![no_std]
+#![cfg_attr(not(feature = "std"), no_std)]
+#![cfg_attr(not(feature = "std"), no_main)]
```

Comment:
Is this temporary or why would we want `std` examples?

Reply:
We don't need `std` example ever. However, this is just to get `rust-analyzer` in vscode to accept the examples without errors.

Reply:
You should be able to configure rust-analyzer to not need that. The configuration you're looking for is `rust-analyzer.cargo.target`. I don't know how it's called in VSCode though. You should set it to `thumbv7em-none-eabi`.

Reply:
Hmm, I tried that but then I get failing build scripts errors in vscode, in the "ring" dependency.

Reply:
What is vscode trying to build, does it give you a command? And do the examples work if you build them manually?
```
./deploy.py --board=nrf52840dk_opensk --console_test
```

Reply:
The can be built no problem. The "error" was rather that rust-analyzer indicated errors in the examples without the attributes in vscode. Checked just again: Seems to work now without this workaround :)